virttest/utils_v2v.py

```diff
@@ -869,3 +935,26 @@ def cleanup_constant_files(params):
                 params.get("vpx_passwd_file")]
 
     map(os.remove, [x for x in tmpfiles if x and os.path.isfile(x)])
+
+
+def get_vddk_thumbprint(host, password, prompt=r"[\#\$]"):
+    """
+    Get vddk thumbprint from VMware vCenter
+
+    :param host: Hostname or IP address
+    :param password: Password
+    :param prompt: Shell prompt (regular expression)
+    """
+    cmd = 'openssl x509 -in /etc/vmware-vpx/ssl/rui.crt -fingerprint -sha1 -noout'
+
+    try:
```

Comment:
no need try ... except since you just raise exception when happen

Reply:
Done